packages/patterns-react/src/patterns/SimpleLongForm/SimpleLongForm.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import { settings } from 'carbon-components';
+import { featureFlag } from '@carbon/ibmdotcom-utilities';
+import { SIMPLELONGFORM } from '../../internal/FeatureFlags';
+import JumpLink from './JumpLink';
+import CardLink from './CardLink';
```

Comment:
Are we swapping out with the global CardLink?

Reply:
I can do it. How can I pull CardLink? I don't have it in local package.

Reply:
`import { CardLink } from '@carbon/ibmdotcom-react';`

You'll also need to add the `CARDLINK` feature flag locally.

cc: @photodow

Reply:
alright, that's simple :)

Reply:
Do a `yarn link` for now, but note that the CardLink component itself isn't available in an official release yet. You may need to update package.json to point to the latest alpha of `@carbon/ibmdotcom-react` in the patterns-react package.

Reply:
Swapped out with global CardLink by [this commit](https://github.com/carbon-design-system/ibm-dotcom-library/pull/561/commits/c36ac9cf10ebdb0789cdbc4377ec255373df23c3)

>Note: CardLink doesn't work with Dark theme yet.
@photodow setting card background color to $ui-01 might helps.
 ` background-color: $ui-01;`